src/gluonts/evaluation/metrics.py

```diff
@@ -87,7 +87,7 @@ def coverage(target: np.ndarray, forecast: np.ndarray) -> float:
 
         coverage = mean(Y < \hat{Y})
     """
-    return np.mean(target < forecast)
+    return np.mean(target < forecast, dtype=float)
```

Comment:
Is this change related?

Reply:
```
src/gluonts/evaluation/metrics.py:90: error: Incompatible return value type (got "floating[Any]", expected "float")  [return-value]
```

Reply:
I don't see this in the failing mypy job, e.g here: https://github.com/awslabs/gluonts/actions/runs/4355090680/jobs/7611245842

Where do you get this?